README.md

```diff
@@ -1,23 +1,21 @@
 # ExFirebaseAuth 🔥
 
-ExFirebaseAuth is a library that handles ID tokens from Firebase, which is useful for using Firebase's auth solution because Firebase does not have an Elixir SDK for auth themselves. ExFirebaseAuth also comes with some testing utilities that mock and generate ID tokens for your integration tests.
+[![CI](https://github.com/Nickforall/ExFirebaseAuth/actions/workflows/elixir.yml/badge.svg)](https://github.com/Nickforall/ExFirebaseAuth/actions/workflows/elixir.yml)
 
-[More information on how ID tokens work in Firebase Auth](https://firebase.google.com/docs/auth/admin/verify-id-tokens)
+ExFirebaseAuth is an Elixir library to handle ID Tokens from the [Firebase Authentication service](https://firebase.google.com/products/auth).
 
-This library
+This library:
 
-- Keeps track of google's public keys used for signing ID tokens
-- Verifies ID tokens
-- Veries whether the issuer matches your firebase project
+- validates ID Tokens and unpack its user information
+- keeps track of Google public keys used for signing ID Tokens
+- comes with test utils to mock ID Tokens in dev/test environments
+- does not aim to implement the Firebase Admin SDK endpoints
 
-This library does **not**
-
-- Aim to implement Firebase user admin SDK endpoints
+[Read more about Firebase Authentication ID Tokens here.](https://firebase.google.com/docs/auth/admin/verify-id-tokens)
 
 ## Installation
 
-If [available in Hex](https://hex.pm/packages/ex_firebase_auth), the package can be installed
-by adding `ex_firebase_auth` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `ex_firebase_auth` to your list of dependencies in `mix.exs`:
```

Comment:
Here I kept the information already in place, repositioning and highlighting important things (the dos and don'ts of the library - I liked your approach, btw).